.github/workflows/release.yml

```
@@ -48,6 +48,10 @@ jobs:
       commit_message: 'Update examples link in index.html.markdown for ${{ github.event.inputs.version_number }} release'
       user_email: 'releasebot@mongodb.com'
       user_name: 'releasebot'
+    secrets:
+      remote: https://svc-apix-bot:${{ secrets.APIX_BOT_PAT }}@github.com/${{ github.repository }}  
+      gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+      passphrase: ${{ secrets.PASSPHRASE }}     
```

After having a second look, in the release process for creating commits we should use the same gpg values and user_email/user_name so that we have verified commits (no need to have these values configurable). `secrets.GPG_PRIVATE_KEY` will be used in the GoReleaser step.
```suggestion
    secrets: inherit   
```

---

Yes for commits to be signed we will have to use the APIX_BOT key and passphrase, changed. Let me know if it's okay now

---

if secrets.GPG_PRIVATE_KEY and  secrets.PASSPHRASE are not used any more we can delete those secrets

---

It's still used when running GoReleaser. For release we will use those, for git operations we will use the new values so that commits are verified